packages/manager/modules/dedicated-universe-components/src/index.js

```diff
@@ -0,0 +1,96 @@
+import angular from 'angular';
+import 'regenerator-runtime/runtime';
```

Comment:
This should be imported in the app not the module. The app should be the one choosing to polyfill or not